src/main/views/list.njk

```diff
@@ -0,0 +1,77 @@
+{% extends "template.njk" %}
+{% block pageTitle %}
+    {{ "Hearings List" | safe }}
+{% endblock %}
+
+{% block beforeContent %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+  {{ govukBackLink({
+    text: "Back",
+    href: "/"
+  }) }}
+{% endblock %}
+
+{% block content %}
+<h2 class="govuk-heading-l govuk-!-margin-bottom-8">{{courtName}} hearing list</h2>
+{% from "govuk/components/table/macro.njk" import govukTable %}
```

Comment:
whack those nunjuck imports to the top of the file 😎

Reply:
Moved to top of file